lib/data-table/table.rb

```diff
@@ -254,7 +254,7 @@ def render_totals
       html = '<tfoot>'
       @total_calculations.each_with_index do |totals_row, index|
         next if totals_row.nil?
-        
+
```

Comment:
unnecessary change?

Reply:
I was just linting.  Should I remove?

Reply:
I typically prefer to make separate commits for linting-only changes, so maybe do that?

Reply:
Sure, I changed it back.